howto/port/install-system-app.md

```diff
@@ -83,4 +83,6 @@ Then create the application with the `amc` command:
 
     amc application create .
 
-After the AMS application is created successfully, the APK is installed as a proper system app. It will run as a system app in the Android container when you start it from a container launched from the newly created application.
+Use the `--vm` flag to create the application in a virtual machine.
```

Comment:
Should we mention that if a specific image is selected that this one must be a VM one when `--vm` is used?

Reply:
I think it's important, otherwise I already imagine we will have plenty of reported bugs for this.